src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/TopXObject.java

```diff
@@ -0,0 +1,107 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays;
+
+import lombok.CustomLog;
+import lombok.Getter;
+import org.betonquest.betonquest.database.Connector;
+import org.betonquest.betonquest.database.QueryType;
+import org.bukkit.Bukkit;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Holds data of a ranking.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.